### PR TITLE
Datashader tutorial

### PR DESCRIPTION
**Pull request by emmanuelle**

### CODE REVIEW

**Comment by emmanuelle** [comment]
The CI failure seems to be unrelated...

**Comment by nicolaskruchten** [comment]
There is a doc build failure as well though, something is missing in requirements.txt: https://29942-14579099-gh.circle-artifacts.com/0/doc/build/failures/datashader

File: doc/python/datashader.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.7.3
+  plotly:
+    description: How to use datashader to rasterize large datasets, and visualize
+      the generated raster data with plotly.
+    display_as: scientific
+    language: python
+    layout: base
+    name: Using datashader with plotly
```
Comment (nicolaskruchten) [inline doc/python/datashader.md]: I would call this `Plotly and Datashader`

File: doc/python/datashader.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    order: 21
+    page_type: u-guide
+    permalink: python/datashader/
+    thumbnail: thumbnail/datashader.jpg
```
Comment (nicolaskruchten) [inline doc/python/datashader.md]: does this thumbnail exist yet?

Comment (nicolaskruchten) [reply doc/python/datashader.md]: you can reuse `thumbnail/heatmap_colorscale.jpg`

**Comment by nicolaskruchten** [comment]
💃

**Comment by nicolaskruchten** [comment]
Looks awesome, just what we needed :)

**Comment by nicolaskruchten** [comment]
Ah one thing though: can you add links *to* this page from the relevant existing pages? like imshow, heatmap, the webgl page, the mapbox layers page maybe?

**Comment by emmanuelle** [comment]
The datashader icon exists already

**Comment by nicolaskruchten** [comment]
OK let's merge this!

**Pull request merged**